requirements/base.in

```diff
@@ -8,3 +8,5 @@ pytz
 six
 celery
 edx-django-utils
+openedx-events
```

Comment:
You may want to recompile with a min-constraint in order to ensure the recursion-prevention fix is picked up wherever this library is used.

```suggestion
openedx-events>=9.5.1
```

Reply:
ready